taqueria-vscode-extension/src/lib/gui/web/WebUI.ts

```diff
@@ -126,4 +126,4 @@ export const createVscodeWebUiHtml = ({
string"?e:e&&"friendlyName"in e&&e.friendlyName||Pf,i=typeof e=="string"?!1:e&&"respectConnection"in e&&e.respectConnection||!1,s=function(o,l,a){if(o==null||new.target!==void 0)throw new Error(`No registration for interface: \'${s.friendlyName}\'`);if(l)he.defineProperty(o,l,s,i);else{const u=he.getOrCreateAnnotationParamTypes(o);u[a]=s}};return s.$isInterface=!0,s.friendlyName=r==null?"(anonymous)":r,n!=null&&(s.register=function(o,l){return n(new S1(o,l!=null?l:s))}),s.toString=function(){return`InterfaceSymbol<${s.friendlyName}>`},s},inject(...e){return function(t,n,r){if(typeof r=="number"){const i=he.getOrCreateAnnotationParamTypes(t),s=e[0];s!==void 0&&(i[r]=s)}else if(n)he.defineProperty(t,n,e[0]);else{const i=r?he.getOrCreateAnnotationParamTypes(r.value):he.getOrCreateAnnotationParamTypes(t);let s;for(let o=0;o<e.length;++o)s=e[o],s!==void 0&&(i[o]=s)}}},transient(e){return e.register=function(n){return Is.transient(e,e).register(n)},e.registerInRequestor=!1,e},singleton(e,t=T1){return e.register=function(r){return Is.singleton(e,e).register(r)},e.registerInRequestor=t.scoped,e}}),$1=he.createInterface("Container");he.inject;const T1={scoped:!1};class bt{constructor(t,n,r){this.key=t,this.strategy=n,this.state=r,this.resolving=!1}get $isResolver(){return!0}register(t){return t.registerResolver(this.key,this)}resolve(t,n){switch(this.strategy){case 0:return this.state;case 1:{if(this.resolving)throw new Error(`Cyclic dependency found: ${this.state.name}`);return this.resolving=!0,this.state=t.getFactory(this.state).construct(n),this.strategy=0,this.resolving=!1,this.state}case 2:{const r=t.getFactory(this.state);if(r===null)throw new Error(`Resolver for ${String(this.key)} returned a null factory`);return r.construct(n)}case 3:return this.state(t,n,this);case 4:return this.state[0].resolve(t,n);case 5:return n.get(this.state);default:throw new Error(`Invalid resolver strategy specified: ${this.strategy}.`)}}getFactory(t){var n,r,i;switch(this.strategy){case 1:case 2:return t.getFactory(this.state);case 5:return(i=(r=(n=t.getResolver(this.state))===null||n===void 0?void 0:n.getFactory)===null||r===void 0?void 0:r.call(n,t))!==null&&i!==void 0?i:null;default:return null}}}function Tf(e){return this.get(e)}function I1(e,t){return t(e)}class A1{constructor(t,n){this.Type=t,this.dependencies=n,this.transformers=null}construct(t,n){let r;return n===void 0?r=new this.Type(...this.dependencies.map(Tf,t)):r=new this.Type(...this.dependencies.map(Tf,t),...n),this.transformers==null?r:this.transformers.reduce(I1,r)}registerTransformer(t){(this.transformers||(this.transformers=[])).push(t)}}const P1={$isResolver:!0,resolve(e,t){return t}};function Po(e){return typeof e.register=="function"}function O1(e){return Po(e)&&typeof e.registerInRequestor=="boolean"}function If(e){return O1(e)&&e.registerInRequestor}function R1(e){return e.prototype!==void 0}const _1=new Set(["Array","ArrayBuffer","Boolean","DataView","Date","Error","EvalError","Float32Array"
```

Comment:
Should this even be committed?

Reply:
We can ignore it, but then we need to re-generate it before each TVsCE build.
Good idea, but I suggest we don't do it now.